### ISSUE TITLE
[Feature Request] Tabbed Bookmarks Group

### ISSUE THREAD

**Issue by ghost-of-cerberus** [joelacus/Cobalt#13]
Hi @joelacus! Greatly appreciate all the updates, Cobalt runs better than ever. 😃

Would it be possible to allow the current bookmark group widget to have an option to show the different groups as tabs? I think this would be better from an organisational and focus standpoint. Multiple bookmark groups and workspaces could be used to isolate bookmarks into their own sections but seems a bit much and less productive than one area which can be tabbed through.

If not possible with the current bookmark group widget, a separate tabbed bookmark groups widget would also be welcome.

Again, thanks for such a wonderful extension!

**Comment by joelacus** [comment]
> Greatly appreciate all the updates, Cobalt runs better than ever. 😃

I'm happy to hear that :)

> Would it be possible to allow the current bookmark group widget to have an option to show the different groups as tabs?

That's a good idea, I hadn't thought about that actually. I should be able to add an option for the Bookmark Group widget to display multiple groups as tabs, no need for an extra widget. I'll work on that for the next update.

> Again, thanks for such a wonderful extension!

You're welcome, thanks for sticking with it and helping me make it better :)

**Comment by joelacus** [comment]
Hi. I've just published version 0.9.0 with this feature! Just go to the widget editor for your bookmark group and enable "Show Groups As Tabs". I added a few tab style options too. It will also remember which tab is selected. You'll still be able to move bookmarks from one group to another by dragging the bookmark and hovering over another tab. Let me know if you have any issues or other suggestions, thanks! :)
